aptos-move/framework/src/module_metadata.rs

```diff
@@ -195,6 +195,30 @@ pub fn get_metadata_from_compiled_module(
     }
 }
 
+/// Extract metadata from a compiled module, upgrading V0 to V1 representation as needed.
+pub fn get_metadata_from_compiled_script(
```

Comment:
This is mostly a copy paste of the existing function `get_metadata_from_compiled_script`. In the API types there is a unifying trait for modules and scripts called `Bytecode` that could help eliminate this duplication, since all we need is a way to access the metadata, but we'd have to move that trait outside of the API types and into somewhere more reasonable for the framework to access.

I looked to see if there is another trait similar to this but I can't find one.

Reply:
Can you leave this as a comment here to make this clear?

Reply:
Can do